README.md

```diff
@@ -456,7 +457,7 @@ import pytz
 from datetime import datetime
 
 conn = trino.dbapi.connect(
-    experimental_python_types=True,
+    legacy_primitive_types=False,
```

Comment:
This doesn't really make sense as this is the default. I would just remove this code fragment

Reply:
removed